### DISCUSSION TITLE
Rinkeby ETH Faucet

### DISCUSSION THREAD

**Discussion by Robcar-10** [smartcontractkit/full-blockchain-solidity-course-py#954] [Q&A] [answered]
Been trying to get some test ETH from the Rinkeby Faucet. I tweeted the request and when I click either 3/7.5 or 18.75 ETH I get this error: insufficient funds for gas * price + value - has anyone encountered this error? 

Could anyone help? Test Adress is: 0x61F66211B0B503a7eCBCCb4f1217aaadA88B5fdd 

TIA!

**Comment by cromewar** [comment, marked as answer]
The official faucet tends to not work time to time, you can use this one: https://faucets.chain.link/
https://rinkeby.etherscan.io/tx/0x1b0766cbd9745f8d2e430fbd7416349b5a50f8ad0aa2ee5268765925682ffb8f

**Reply by Robcar-10** [reply]
Thank you sir! Appreciate your help and sending me some test ETH

**Comment by n4n0b1t3** [comment]
This one works fine for weeks
https://www.rinkebyfaucet.com/
Most faucets give not more than 0.1 ETH nowadays. You will just have to adapt your code to transfer less. 0.1ETH will last a while when you just use it wisely.
